backend/src/main/java/ca/bc/gov/hlth/accounttransfer/security/AudienceValidator.java

```diff
@@ -0,0 +1,45 @@
+package ca.bc.gov.hlth.accounttransfer.security;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.security.oauth2.resource.OAuth2ResourceServerProperties;
+import org.springframework.security.oauth2.core.OAuth2Error;
+import org.springframework.security.oauth2.core.OAuth2ErrorCodes;
+import org.springframework.security.oauth2.core.OAuth2TokenValidator;
+import org.springframework.security.oauth2.core.OAuth2TokenValidatorResult;
+import org.springframework.security.oauth2.jwt.Jwt;
+import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.springframework.security.oauth2.core.oidc.IdTokenClaimNames.AUD;
+
+@Component
+public class AudienceValidator implements OAuth2TokenValidator<Jwt> {
+
+    private static final Logger logger = LoggerFactory.getLogger(AudienceValidator.class);
+
+    private static final OAuth2Error error = new OAuth2Error(OAuth2ErrorCodes.INVALID_REQUEST, "The " + AUD + " claim is not valid",
+            "https://tools.ietf.org/html/rfc6750#section-3.1");
+
+
+    private static final String AUDIENCE = "LDAP-ACCOUNT-TRANSFER-API";
```

Comment:
Why is this not specified in spring security props, ...jwt.audience?

Reply:
Just harded coded it in initial setup and forgot to move it. 
Fixed